package.json

```diff
@@ -33,5 +33,9 @@
     "react": "^16.7.0",
     "react-dom": "^16.7.0",
     "react-scripts": "2.1.3"
+  },
+  "dependencies": {
```

Comment:
I've just noticed that I've added libs to "dependencies" instead of "devDependencies". Should I place them in "devDependencies" as well ?

Reply:
I think GraphQL-ws should be in deps as it's used in runtime